package.json

```diff
@@ -65,72 +65,81 @@
     ]
   },
   "peerDependencies": {
-    "@react-three/drei": ">=8.16",
-    "@react-three/fiber": ">=7.0",
-    "react": ">=17.0",
-    "react-dom": ">=17.0",
-    "suspend-react": "0.0.8",
-    "three": ">=0.138",
-    "three-stdlib": ">=2.8"
+    "@react-three/drei": ">=9.78",
+    "@react-three/fiber": ">=8.13",
+    "@react-three/postprocessing": ">=2.14",
+    "postprocessing": ">=6.32",
+    "react": ">=18.0",
+    "react-dom": ">=18.0",
+    "suspend-react": "0.1.3",
+    "three": ">=0.154",
+    "three-stdlib": ">=2.23"
```

Comment:
We should note this as breaking change via [commit message](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer).